modules/nf-core/bedtools/nuc/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: bedtools_nuc
+description: Profiles the nucleotide content of intervals in a fasta file.
+keywords:
+  - bed
+  - nucBed
+  - bedtools
+  - GC content
+  - AT content
+  - nucleotide content
+  - intervals
+tools:
+  - bedtools:
+      description: A set of tools for genomic analysis tasks, specifically enabling
+        genome arithmetic (merge, count, complement) on various file types.
+      homepage: https://bedtools.readthedocs.io
+      documentation: https://bedtools.readthedocs.io/en/latest/content/tools/nuc.html
+      doi: "10.1093/bioinformatics/btq033"
+      licence: ["MIT"]
+      identifier: biotools:bedtools
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
+    - fasta:
+        type: file
+        description: Genome reference
+        pattern: "*.{fa,fasta}"
+    - bed:
```

Comment:
Maybe `bed` is not the correct name here since it can also be a `gff` or `vcf` file, can you think of a better name?

Reply:
Do you think `intervals` would be a better fitting name? The tool calculates nucleotide statistics for a given interval, which is given by `bed` and `gff` files. However one could argue that the vcf file doesn't give an interval, but I assume the tool is infering the intervals from the information available within the vcf file.

Reply:
Yeah that seems fine in this case :)